app/views/users/show.html.erb

```diff
@@ -1,39 +1,14 @@
-<div class="users-list">
- <div class="single-user">
-<div class="user-image"></div>
-<div class="user-details">
-<div class="user-username"><%= @user.name %></div>
-<div class="user-posts-count">number of posts: 
-<% if @user.posts_counter.to_i > 0 %>
-  <%= @user.posts_counter %>
-<% else %>
-  0
+<%= render partial: 'partials/user_partial', locals: { user: @user } %>
+
+<div >
+<p>Bio
+<p>
+<%= @user.bio%>
+</p>
+</p>
+
+<% @recent_post.each do |recent| %>
+<%= render partial: 'partials/post_partial', locals: { post: recent}  %>
```

Comment:
- [ ] Great job woking on it but please make sure to have the same wireframe as I asked previously  it would be great if you make you layout the same as the given wireframe no styles at this moment but the wireframe it should be the same so kindly work on it to have it similar to the given one 🔥❤🚀

- Yours
![image](https://user-images.githubusercontent.com/58553711/231931224-7c9133bc-eeb1-49f2-9526-616f4ea0b87d.png)

- Given
![image](https://user-images.githubusercontent.com/58553711/231931285-d7f2b070-dee6-4ec6-8169-b87bd56f0885.png)